networkx/algorithms/triads.py

```diff
@@ -224,6 +249,17 @@ def is_triad(G):
     -------
     istriad : boolean
        Whether G is a valid triad
+
+    Examples
+    --------
+    >>> G = nx.DiGraph()
+    >>> G.add_edges_from([(1, 2), (2, 3), (3, 1)])
```

Comment:
A very minor consideration, but IMO it's valuable to keep the docstring examples as concise as possible so readers can focus on the important part (i.e. `is_triad` in this case).
```suggestion
    >>> G = nx.DiGraph([(1, 2), (2, 3), (3, 1)])
```

It looks like this comment applies to many of the other examples as well.

Reply:
Yes Sir! you mentioned this in a previous PR as well. Its my force of habit 😅. I will make sure to edit the others as well. Thanks!

Reply:
Yes Sir! you mentioned this in a previous PR as well. Its my force of habit 😅. I will make sure to edit the others as well. Thanks!